bika/lims/browser/aggregatedanalyses/aggregatedanalyses.py

```diff
@@ -58,15 +58,36 @@ def __init__(self, context, request, **kwargs):
             }
         self.review_states = [
             {'id': 'default',
-             'title':  _('Results pending'),
+             'title': _('Assignment pending'),
+             'transitions': [{'id': 'submit'},
+                             {'id': 'cancel'},
+                             # {'id': 'assign'}
+                             ],
+             'contentFilter': {
+                 'review_state': ['sample_received'],
+                 'cancellation_state': 'active', },
+             'columns': ['AnalysisRequest',
+                         'Worksheet',
+                         'Service',
+                         'Result',
+                         'Uncertainty',
+                         'Partition',
+                         'Method',
+                         'Instrument',
+                         'Analyst',
+                         'state_title',
+                         ]
+             },
+            {'id': 'results_pending',
+             'title': _('Results pending'),
              'transitions': [{'id': 'sample'},
                              {'id': 'submit'},
                              {'id': 'cancel'},
                              # {'id': 'assign'}
                              ],
-             'contentFilter': {'review_state': [
-                'sample_received', 'assigned', 'attachment_due'],
-                'cancellation_state': 'active', },
+             'contentFilter': {
+                 'review_state': ['assigned', 'attachment_due'],
```

Comment:
Please note that `assigned` state belongs to `worksheetanalysis_workflow` (see my comment above), so use the variable `worksheetanalysis_review_state` for filtering by `assigned/unassigned` states. Since we want this view to display those analyses that are result pending, no matter if they are assigned or not, just do the following and don't consider 'assigned/unassigned':
```python
'review_state': ['sample_received', 'attachment_due']
```

Reply:
As before, please add `DetectionLimit` and `Specifications` columns at least, in same order as in `AnalysesView`. In "Assignment pending" state it was not necessary to display the columns "Worksheet" and "Analyst", but in this state "Results pending" is mandatory.